packages/cxl-ui/src/components/cxl-popover-card.js

```diff
@@ -0,0 +1,193 @@
+import { html, LitElement, render } from 'lit';
+import { customElement, property, query } from 'lit/decorators';
+import '@conversionxl/cxl-lumo-styles';
+import cxlPopoverCardStyles from '../styles/cxl-popover-card-css';
+import CXLSingleMultiversionCardDisplay from "./partials/cxl-single-multiversion-card-display";
+
+@customElement('cxl-popover-card')
+export class CXLPopoverCardElement extends LitElement {
+  @property({ type: String })
+  apiUrl = 'https://cxl.fake/playbooks/1';
+
+  @property({ type: String })
+  href = 'https://google.com';
+
+  @property({ type: String })
+  text = 'Google';
+
+  @property({ type: Number })
+  userId = 1234;
```

Comment:
I don't think we should hardcode some temp values. Can't we just make these properties empty strings or 0?

cc @lkraav

Reply:
@pawelkmpt @lkraav @anoblet 

Let me explain why _nulls/empty strings_ are not used here:
1) If you pass an empty string (or null) to apiUrl you'll get an error. Instead, you can use a working default value similarly as used in the common Null Object Pattern to avoid the default error (or redundant error check).
2) here apiUrl is not actually even a fake, thanks to the  [MSW Storybook Addon](https://storybook.js.org/addons/msw-storybook-addon) which we've been talking about with @anoblet and confirmed to use it in the stories) we can output real JSON data under this URL.
3) default values make it clear at first glance what specific property should look like in case any new developer will have to deal with the component (especially that currently there is no documentation regarding CXL Web Components).

But If you still want to avoid default values for properties for whatever reason I can remove them.

Reply:
> 1. If you pass an empty string (or null) to apiUrl you'll get an error. Instead, you can use a working default value similarly as used in the common Null Object Pattern to avoid the default error (or redundant error check).

Use sanity checks and if specific conditions are not met just do nothing but displaying original URL.

> 2. here apiUrl is not actually even a fake, thanks to the MSW Storybook Addon which we've been talking about with @anoblet and confirmed to use it in the stories) we can output real JSON data under this URL.

It's definitely cool for prototyping and development but for me it looks like using "lorem ipsum" on the live site.

> 3. default values make it clear at first glance what specific property should look like in case any new developer will have to deal with the component (especially that currently there is no documentation regarding CXL Web Components).

You can use comments with examples.

> But If you still want to avoid default values for properties for whatever reason I can remove them.

Default values are good in some cases, eg. default font color or placeholder text, but here they look like a leftover from testing. They should be removed and relevant conditions added so component will not throw an error if eg. API URL is missing.

Do you agree @lkraav ?